packages/procaptcha-common/src/providers.ts

```diff
@@ -0,0 +1,51 @@
+import { loadBalancer } from "@prosopo/load-balancer";
+import type {
+	ProcaptchaClientConfigOutput,
+	RandomProvider,
+} from "@prosopo/types";
+import { at } from "@prosopo/util";
+
+export const getRandomActiveProvider = async (
+	config: ProcaptchaClientConfigOutput,
+): Promise<RandomProvider> => {
+	const randomIntBetween = (min: number, max: number) =>
+		Math.floor(Math.random() * (max - min + 1) + min);
+
+	// TODO maybe add some signing of timestamp here by the current account and then pass the timestamp to the Provider
```

Comment:
I don't think this matters tbh. We're just spreading frontend load to the backend, we don't need to verify if it's been done properly bc the majority of traffic will have

Reply:
This is just copying over a comment from @forgetso so leaving in for now